Userland/Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -255,7 +257,7 @@ void ResourceLoader::load(LoadRequest& request, Function<void(ReadonlyBytes, Has
             if (maybe_file.is_error()) {
                 log_failure(request, maybe_file.error());
                 if (error_callback)
-                    error_callback(DeprecatedString::formatted("{}", maybe_file.error()), maybe_file.error().code());
+                    error_callback(DeprecatedString::formatted("{}", maybe_file.error()), {});
```

Comment:
The 'error code is optional' API is already a massive PITA, let's not add more. 500 maybe?

Reply:
Sounds reasonable